amethyst_renderer/src/formats/texture.rs

```diff
@@ -70,6 +70,15 @@ impl TextureMetadata {
         }
     }
 
+    /// Creates texture metadata for `Srgb` texture but without texture filtering. This is usually
+    /// the case for color textures for 2D sprites and pixel art textures.
+    ///
+    /// For the values of all the other fields please refer to the documentation of the respective
+    /// field.
+    pub fn srgbScale() -> Self {
```

Comment:
You must use snake_case for functions, according to Rust's formatting guidelines.
`srgb_nearest` maybe?